addons/hr_employee_hourly_wage/security/hr_employee_hourly_wage_security.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="group_employee_hourly_wage_manager" model="res.groups">
+            <field name="name">Administrator</field>
+            <field name="category_id" ref="base.module_category_services_employee_hourly_wage"/>
+            <field name="implied_ids" eval="[(4, ref('hr.group_hr_user'))]"/>
+            <field name="users" eval="[(4, ref('base.user_root')), (4, ref('base.user_admin'))]"/>
+        </record>
+    </data>
+</odoo>
```

Comment:
Missing new line at the end of the file. This can be avoided by enabling the following setting in VS Code
![image](https://user-images.githubusercontent.com/88150098/188577421-f32f52c8-738d-4de8-9ca3-b43d0297ee9e.png)